jscomp/repl.js

```diff
@@ -73,7 +73,7 @@ prepare()
 
 console.log(`playground : ${playground}`)
 
-var includes = [`stdlib`, `runtime`, `others`].map(x => path.join(__dirname, x)).map(x => `-I ${x}`).join(` `)
+var includes = [`stdlib`, `runtime`, `others`, `ext`].map(x => path.join(__dirname, x)).map(x => `-I ${x}`).join(` `)
 
```

Comment:
why `ext` is needed here?

Reply:
Another experiment. Let me double-check if it's needed.

Reply:
You are right, it's not needed.